src/components/LessonInfoDisplay/LessonInfoDisplay.scss

```diff
@@ -1,95 +1,156 @@
 @import "../../pages/Index/index.scss";
 
-.LessonInfo {
+.lesson-info {
   background-color: $assignments-darker-color;
   font-family: $assignments-roboto-font;
   width: 70%;
-  height: 23vw;
-  padding: 20px;
+  height: 25vw;
   border: 1px solid #4d4d4c;
   border-radius: 2em;
   position: relative;
   margin: auto;
   margin-top: 8em;
   z-index: 1;
+  overflow: hidden;
 }
 
-.InfoDisplay {
+.info-display {
   display: flex;
   flex-direction: row;
   flex-wrap: wrap;
-  height: 18vw;
+  overflow: hidden;
 }
 
-.Column {
-  height: 17vw;
-  margin: auto;
+.column {
+  float: left;
+  height: 25vw;
+  width: 18vw;
+  background-color: black;
+  border-top-left-radius: 2em;
+  border-bottom-left-radius: 2em;
+  overflow:auto;
 }
 
-.WordGroupImage {
-  height: 13vw;
-  width: 13vw;
+.word-group-display {
+  height: 5vw;
+  width: 16vw;
   background-color: $assignments-light-purple;
-  color: black;
+  color: #4D4C4C;
   border: 1px solid black;
-  border-radius: 100%;
+  border-radius: 2em;
   margin: auto;
-  margin-bottom: 10px;
+  margin-left: 1.3em;
+  margin-top: 1em;
+  overflow: hidden;
 }
 
-.WordGroupName {
-  font-size: 1.3em;
-  font-weight: normal;
+.lesson-group-name {
+  margin-top: 5%;
+  padding:auto;
+  font-size: 2em;
+  font-weight: bold;
   text-transform: uppercase;
-  color: black;
+  color: $assignments-font-color;
 }
 
-.LessonDataTitle {
+.word-display {
+  height: 16vw;
   width: 15vw;
+  background-color: white;
+  border-radius: 1em;
+  margin: auto;
+  margin-top: 1em;
+  display: block;
+  overflow: hidden;
+  margin-left: 1.8em;
+}
+.grey-box, .purple-box {
   height: 3vw;
-  border: 1px solid black;
-  border-top-left-radius: 8px;
-  border-top-right-radius: 8px;
-  background-color: lightgray;
+  background-color: #C4C4C4;
```

Comment:
what color is this and does it need a variable? Ik gene made some variables for light gray in his branch though